src/thunderbots/software/ai/navigator/rrt/rrt.cpp

```diff
@@ -39,10 +40,7 @@ std::vector<std::unique_ptr<Primitive>> RRTNav::getAssignedPrimitives(
         }
         else
         {
-            // TODO: Throw a proper exception here
-            // https://github.com/UBC-Thunderbots/Software/issues/16
-            std::cerr << "Error: Unrecognized Intent given to navigator" << std::endl;
-            exit(1);
+             throw std::invalid_argument("Error: Unrecognized Intent given to navigator STAAP OR ELSE");
```

Comment:
Would be good to also include the name of the unrecognized Intent in the message. Makes it quicker for people to see if the error is just a typo in the name or something without having to do a full debugging session. (Can put that in place of `STAAP OR ELSE` :stuck_out_tongue_winking_eye: )

Reply:
As a general rule, having entertaining error messages is amusing initially, but can get a bit aggravating and unhelpful over time (`#warning` still triggers me).